xgitguard/common/configs_read.py

```diff
@@ -227,6 +244,60 @@ def read_stop_words(self, file_name="stop_words.csv"):
         self.stop_words = [item for sublist in self.stop_words for item in sublist]
         # logger.debug(f"Total Stop Words: {len(self.stop_words)}")
 
+    def read_search_paths(self, file_name):
+        """
+        Read the given search paths  csv file in config path
+        Set the Class Variable for further use
+        params: file_name - string
+        """
+        logger.debug("<<<< 'Current Executing Function' >>>>")
+        # Loading secondary keywords from secondary keywords file
+        self.search_paths_file = os.path.join(self.config_dir, file_name)
+        self.search_paths = read_csv_file(
+            self.search_paths_file, output="list", header=0
+        )
+        self.search_paths = [item for sublist in self.search_paths for item in sublist]
+        # logger.debug(f"search_paths: {self.search_paths}")
+
+    def read_search_files(self, file_name):
+        """
+        Read the given search paths  csv file in config path
+        Set the Class Variable for further use
+        params: file_name - string
+        """
+        logger.debug("<<<< 'Current Executing Function' >>>>")
+        # Loading secondary keywords from secondary keywords file
```

Comment:
comment is pointing to secondary keywords, let's change it to respective ones.

Reply:
updated comment statement